File: Python/uberBlack.py
from car import Car


class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, passenger, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver, passenger)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: JS/UberBlack.js
import Car from Car

export class UberBlack extends Car{
  constructor(license, driver, passenger, typeCarAccepted, seatsMaterial){
    super(license, driver, passenger);
    this.typeCarAccepted = typeCarAccepted;
    this.seatsMaterial = seatsMaterial;
  }
}